src/dfa/LG_Actions2.cpp

```diff
@@ -465,7 +465,7 @@ Ret:  T_end = token.end;
 int   LG_ParseActions::START_GRM ()
 {
    int rc = 0;
-   short p = 0, a = 0;
+   short p = 0; //, a = 0;
```

Comment:
This review tool has some unusual quirks.  For one, it won't allow comments on unchanged lines, as far as I can tell... so I comment here.

If 'rc' is set to 0 and unchanged, it could be removed and function made to return 'void'.  This will generate better code; the compiler will not need to set a register to 0 before the return.

If a is not needed, please remove it from the code.

Reply:
This one I agree that's ok to remove.